Script Examples/JavaScript/AccessibleFromAutomationTest.js

```diff
@@ -73,7 +73,6 @@ logInfo(" activate "+__LINE__, "State", State);
 logInfo(" activate "+__LINE__, "Command", Command);
 //logInfo(" activate "+__LINE__, "DateTime", DateTime);
 logInfo(" activate "+__LINE__, "LocalTime", LocalTime);
-logInfo(" activate "+__LINE__, "StringUtils", StringUtils);
 logInfo(" activate "+__LINE__, "URLEncoder", URLEncoder);
 logInfo(" activate "+__LINE__, "FileUtils", FileUtils);

```

Comment:
Just to make sure it will be working after https://github.com/openhab/openhab-core/pull/1441 has been merged you should replace other `org.apache.commons` libs too:

Remove:
- FileUtils
- FilenameUtils

Add:
- java.nio.file.Files
- java.nio.file.Path
- java.nio.file.Paths